docs/validation/6_multirotor_sims.tex

```diff
@@ -0,0 +1,216 @@
+\subsection{Simulation Methodology}
+In order to reliably simulate multirotor parameters, \textit{xcopterCalc}, a free tool provided by \textit{eCalc}, is utilized. This tool is widely used throughout the multirotor design community --- \textit{eCalc} is partnered with several prominent multirotor manufacturers (such as LeoMotion, DualSky, and more). Through the use of \textit{xcopterCalc}, differing model parameters (such as battery size, payload weight, and propeller selection) can be tweaked in order to obtain a reasonable estimate for the multirotor's actual flight performance. 
```

Comment:
Hmm, maybe a nitpick but the intro sentence sounds awkward. Perhaps reorder it to be in active voice with fewer commas?

Reply:
Yeah, I agree, not my finest work :). I'll update it.